l10n_es_mis_report/data/mis_report_pyg_abreviated.xml

```diff
@@ -143,7 +143,7 @@
         <field
             name="description"
         >9. Imputación de subvenciones de inmovilizado no financiero y otras</field>
-        <field name="expression">-balp[40900%]</field>
+        <field name="expression">-balp[7460%]</field>
```

Comment:
Resulta raro que en los otros planes, aquí venía la 7460 ya. ¿Sabéis por qué estaba puesta la 4090? cc @AlbertCabedo @acysos

Reply:
Me atrevería a decir que puede ser un error de copia/pega de:

Reply:
Sí, sí, es cierto.